ts-closure-transform/test/serialization/generator.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from 'chai';
+import { serialize, deserialize } from '../../../serialize-closures/src';
+
+function roundtrip<T>(value: T): T {
+  return deserialize(serialize(value));
+}
+
+export function* myGenerator() {
+  yield 42;
+  yield 84;
+}
+
+export function simpleGenerator() {
+  expect(myGenerator().next()).to.equal(42);
```

Comment:
This should be `expect(myGenerator().next()).to.deep.equal({ value: 42, done: false });`

Reply:
Good catch, agree.